src/live_restore/live_restore_fs.c

```diff
@@ -1198,6 +1204,21 @@ __wt_live_restore_fh_import_extents_from_string(
     if (!F_ISSET(S2C(session), WT_CONN_LIVE_RESTORE_FS))
         return (0);
 
+    bool extent_string_empty = extent_str == NULL || strlen(extent_str) == 0;
+
+    /*
+     * Once we're in the clean up stage or later all data has been migrated across to the
+     * destination. There's no need for hole tracking and therefore nothing to import.
+     */
+    WTI_LIVE_RESTORE_STATE state =
+      __wti_live_restore_get_state(session, (WTI_LIVE_RESTORE_FS *)S2C(session)->file_system);
+    if (WTI_LIVE_RESTORE_MIGRATION_COMPLETE(state)) {
```

Comment:
I assume this won't be too hard to migrate to the bitmap change assuming mine goes in first, i'm not fussed about which gets merged first really.

Reply:
Agreed. I'm expecting the only changes will be merge conflict or renaming `extent list` to `bitmap`